### ISSUE TITLE
[Error] tzdata setup broken on fedora-minimal (38)

### ISSUE THREAD

**Issue by wenottingham** [89luca89/distrobox#771]
**Describe the bug**
distrobox is adding /usr/share/zoneinfo to netsharedpath, but isn't actually mounting it, and tzdata is therefore broken (and not easily fixable)

**To Reproduce**
```
[dev]
image=fedora-minimal:latest
init=false
nvidia=false
pull=true
root=false
replace=true
additional_packages=git gcc-c++ glibc-devel autoconf automake joe fedpkg nodejs dnf ipython python3-pip kubernetes-client dos2unix hexedit which
```
```	--version/-V:		show version
[notting@nostromo: ~]$ distrobox-assemble create -f prog/distroboxes/dev.ini 
 - Deleting dev... 
Error: inspecting object: no such container dev
 - Creating dev... 
Trying to pull registry.fedoraproject.org/fedora-minimal:latest...
Getting image source signatures
Copying blob d736d4b924e5 skipped: already exists  
Copying config caaaf3dc9e done  
Writing manifest to image destination
Storing signatures
caaaf3dc9ea78883e86b9c17b935739dda82e21c42c67ceeae0a6da4d773ca2a
Creating 'dev' using image fedora-minimal:latest	 [ OK ]
Distrobox 'dev' successfully created.
To enter, run:

distrobox enter dev

dev
[notting@nostromo: ~]$ distrobox enter dev
Container dev is not running.
Starting container dev
run this command to follow along:

 podman logs -f dev

 Starting container...                  	 [ OK ]
 Installing basic packages...           	 [ OK ]
 Setting up read-only mounts...         	 [ OK ]
 Setting up read-write mounts...        	 [ OK ]
 Setting up host's sockets integration...	 [ OK ]
 Integrating host's themes, icons, fonts...	 [ OK ]
 Setting up package manager exceptions...	 [ OK ]
 Setting up rpm exceptions...           	 [ OK ]
 Setting up sudo...                     	 [ OK ]
 Setting up groups...                   	 [ OK ]
 Setting up users...                    	 [ OK ]
 Integrating host's themes, icons, fonts...	 [ OK ]
 Setting up package manager exceptions...	 [ OK ]
 Setting up rpm exceptions...           	 [ OK ]
 Setting up sudo...                     	 [ OK ]
 Setting up groups...                   	 [ OK ]
 Setting up users...                    	 [ OK ]
 Executing init hooks...                	 [ OK ]

Container Setup Complete!
[notting@dev: ~]$ ls -l /usr/share/zoneinfo
ls: cannot access '/usr/share/zoneinfo': No such file or directory
[notting@dev: ~]$ cat /usr/lib/rpm/macros.d/macros.distrobox 
%_netsharedpath /usr/share/zoneinfo:/etc/localtime:/etc/resolv.conf:/etc/hosts:/etc/host.conf:/tmp:/sys:/proc:/dev:/var/mnt:/var/lib/libvirt:/srv:/run/udev:/run/netconfig/:/run/systemd/resolve/:/run/systemd/users:/run/systemd/sessions:/run/systemd/seats:/run/systemd/journal:/run/media:/run/libvirt:/mnt:/media:/etc/machine-id:/etc/host.conf:/var/log/journal:/var/lib/systemd/coredump:/var/lib/flatpak
[notting@dev: ~]$ cat /proc/mounts | grep zoneinfo
[notting@dev: ~]$ 
```

**Expected behavior**
either:
- /usr/share/zoneinfo bind-mounted from host
*or*
- /usr/share/zoneinfo *not* added to netsharedpath (so I can fix it with rpm)

**Logs**
[logs.txt](https://github.com/89luca89/distrobox/files/11567481/logs.txt)

**Desktop (please complete the following information):**
podman 4.5.0
distrobox version: git head (f008b521ee776a66c36a87848c85d3e3f0f6a3dd)

**Additional context**

**Comment by 89luca89** [comment]
Implemented as of 1cde681369576d3aca9d68834a053422b65a06a7

:-)

**Comment by wenottingham** [comment]
This doesn't seem fixed.

As of 6a4a6b30c4c859319e81d95d886286afd10a94bb , a freshly created box on entry still has empty /usr/share/zoneinfo, with that path in rpm's %_netsharedpath.

**Comment by 89luca89** [comment]
Sorry probably the patch got left out when I pushed
This is implemented now in 536cd158148ed1dd2fca1d6df79c1e3ecd620b94
